packages/core/src/spec/lib/features/FeatureManagerSpec.ts

```diff
@@ -72,13 +72,24 @@ describe('FeatureManager', () => {
       expect(features.applicationClass.imports).toEqual(emptySet);
       expect(features.applicationClass.onCreate).toEqual([]);
       expect(features.applicationClass.variables).toEqual([]);
-      expect(features.buildGradle.dependencies).
-          toEqual((new Set().add('com.google.androidbrowserhelper:androidbrowserhelper:2.0.0')));
+      expect(features.buildGradle.dependencies).toEqual((new Set().
+          add('com.google.androidbrowserhelper:androidbrowserhelper:2.0.0')));
       expect(features.buildGradle.repositories).toEqual(emptySet);
       expect(features.launcherActivity.imports).toEqual(emptySet);
       expect(features.launcherActivity.launchUrl).toEqual([]);
     });
 
+    it('Creates from empty features with alpha features unabled', () => {
+      const manifest = {
+        features: {},
+        fallbackType: 'customtabs',
+        alphaDependencies: {enabled: true},
+      } as TwaManifest;
+      const features = new FeatureManager(manifest);
+      expect(features.buildGradle.dependencies).toEqual((new Set().
+          add('com.google.androidbrowserhelper:androidbrowserhelper:1.4.0-alpha01')));
```

Comment:
A note: this ensures that the **only** dependency in the apps is androidbrowserhelper. Is this indeed what we want to test? If we only want to check if the correct dependency was added, we could do something like this (same thing for the validation above):
```suggestion
      expect(features.buildGradle.dependencies)
          .toContain('com.google.androidbrowserhelper:androidbrowserhelper:1.4.0-alpha01');
```